packages/fluentui/react-northstar/src/themes/teams/components/TextArea/textAreaStyles.ts

```diff
@@ -34,6 +35,13 @@ export const textAreaStyles: ComponentSlotStylesPrepared<TextAreaStylesProps, Te
       color: v.disabledColor,
     }),
 
+    ...(p.autoAdjustHeight && {
+      minHeight: v.minHeightAutoAdjust,
+      height: pxToRem(p.autoHeight),
+      overflow: 'hidden',
+      resize: 'vertical',
```

Comment:
Please would that mean that users could resize the block by themselves? Is that desirable?